### PR TITLE
changelog and pr-template

### PR DESCRIPTION
**Pull request by emanuel-schmid**
analogous to climada_python

### CODE REVIEW

**Comment by emanuel-schmid** [comment]
to be discussed March 15 😄

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- [ ] Read the [Contribution Guide][contrib]
+- [ ] Correct target branch selected (if unsure, select `develop`)
+- [ ] Descriptive pull request title added
+- [ ] Source branch up-to-date with target branch
+- [ ] [Documentation](https://climada-python.readthedocs.io/en/latest/guide/Guide_PythonDos-n-Donts.html#2.--Commenting-&-Documenting) updated
+- [ ] [Tests][testing] updated
+- [ ] [Tests][testing] passing
+- [ ] No new [linter issues][linter]
+- [ ] [Changelog](https://github.com/CLIMADA-project/climada_python/blob/main/CHANGELOG.md) updated
```
Comment (leonie-villiger) [inline .github/pull_request_template.md:17]:
```suggestion
- [ ] [Changelog](https://github.com/CLIMADA-project/climada_petals/blob/main/CHANGELOG.md) updated
```

**Review by leonie-villiger — CHANGES_REQUESTED** [review]
I like! Thanks for adding, but yes let`s see what results from the discussion on 15 March. In case we decide to add the CHANGELOG for python_petals there is one link to be adapted. Apart from that everything looks fine to me.

File: .github/pull_request_template.md (hunk 1 above)
Comment (emanuel-schmid) [reply .github/pull_request_template.md:17]: 😊

**Review by leonie-villiger — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Code freeze date: YYYY-MM-DD
+
+### Description
+
+### Dependency Changes
+
+### Added
+
+- Changelog based on the climada_python CHANGELOG.md (https://keepachangelog.com template) [#72](https://github.com/CLIMADA-project/climada_petals/pull/72)
+- PR Template bassed on climada_python .github/pull_request_template.md [#72](https://github.com/CLIMADA-project/climada_petals/pull/72)
```
Comment (peanutfun) [inline CHANGELOG.md:16]: I think this should be a single entry per PR:
```suggestion
- Changelog and PR description template based on the Climada Core repository [#72](https://github.com/CLIMADA-project/climada_petals/pull/72)
```

**Review by peanutfun — CHANGES_REQUESTED** [review]

**Comment by emanuel-schmid** [comment]
@peanutfun : good catch!

**Comment by leonie-villiger** [comment]
> Thanks! ☺️
> 
> > Thanks for adding, but yes let`s see what results from the discussion on 15 March.
> 
> @leonie-villiger, what was discussed and what was the result? 😅 I think this is good to go and I'll leave you the honors of merging, if your discussion did not come to a different conclusion

The PR was added before _adding a CHANGELOG.md to climada_petals_ was discussed in a climada call. That`s why I wrote "let`s see what results from the discussion". I didn`t participate in this particular climada call but the meeting indicate that the suggestion was approved. Therefore, I will now - with great pleasure - merge this PR ☺️

**Pull request merged**